bridge/client/app/_views/ktb-error-view/ktb-error-view.component.html

```diff
@@ -12,6 +12,10 @@ <h2>Permission denied</h2>
         <h2>Action not permitted</h2>
         <p>User is not allowed to perform this action.</p>
       </ng-container>
+      <ng-container *ngSwitchCase="ServerErrors.CONFIGURATION_SERVICE_NOT_ALLOWED">
+        <h2>Action not permitted</h2>
+        <p>You are not allowed to perform this action if the configuration service is enabled.</p>
```

Comment:
When would that happen? Is the error message correct? I would imply I get this error when the configuration service is **not enabled**?